examples/java/web-service/pom.xml

```diff
@@ -0,0 +1,66 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.mycompany.app</groupId>
+  <artifactId>my-app</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0-SNAPSHOT</version>
+  <name>my-app</name>
+  <url>http://maven.apache.org</url>
+  <dependencies>
+    <!-- https://mvnrepository.com/artifact/org.cdk8s/cdk8s -->
+    <dependency>
+      <groupId>org.cdk8s</groupId>
+      <artifactId>cdk8s</artifactId>
+      <version>0.25.0</version>
+    </dependency>
+    <dependency>
+      <groupId>software.constructs</groupId>
+      <artifactId>constructs</artifactId>
+      <version>2.0.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>29.0-jre</version>
+  </dependency>  
+  <dependency>
+    <groupId>software.amazon.jsii</groupId>
+    <artifactId>jsii-runtime</artifactId>
+    <version>1.5.0</version>
+</dependency>
+  </dependencies>
+  <properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+  <build>
+    <sourceDirectory>src/main/java</sourceDirectory>
+    <testSourceDirectory>src/test/java</testSourceDirectory>
+    <resources>
+      <resource>
+        <directory>imports/src/main/resources</directory>
```

Comment:
Still weird... why are the imports not under `src/main/java/imports`?

Reply:
Hmm, we could probably add a maven step to move them there. They get output to `imports/src/main`. See the output structure here:

https://github.com/eladb/jsii-srcmak/pull/3#issue-434169273

Reply:
```
imports/
  src/
    main/
      java/
        k8s/
        mattermost/
      resources/
        k8s/
        mattermost/
```